src/sage/combinat/q_analogues.py

```diff
@@ -452,10 +451,16 @@ def q_multinomial(seq, q=None, binomial_algorithm='auto'):
 gaussian_multinomial = q_multinomial
 
 
-def q_catalan_number(n, q=None):
+def q_catalan_number(n, m=1, q=None):
     """
     Return the `q`-Catalan number of index `n`.
 
+    INPUT:
+
+    - ``m`` -- optional integer (default: 1) to get the ``m``-Fuss-Catalan numbers
```

Comment:
```suggestion
    - ``m`` -- (optional) the ``m``-Fuss-Catalan number
```

Reply:
not sure I can agree with this suggestion, so I did not perform that

Reply:
An optional argument should not have a default; in terms of the documentation that is. If it has a default, then it is mandatory. Since this is treated as something optional for the user to input, I think it should be marked as such. If we want to say it is a default (and hence essentially required), then I think we should document it as such. Do you agree or have an argument against this?

Reply:
ok, now I understand your point. I made a new tentative